src/load/azext_load/tests/latest/resources/invalid-config-with-zip-artifacts.yaml

```diff
@@ -0,0 +1,22 @@
+displayName: CLI-Test
+testPlan: sample-JMX-file.jmx
+description: 'Test created from az load test command'
+engineInstances: 1
+zipArtifacts:
+   - sample-ZIP-artifact.zip
```

Comment:
Can you put a comment on why this config is invalid?

Reply:
added comments stating reason

Reply:
Thanks!